common/Log.hpp

```diff
@@ -299,7 +299,7 @@ static constexpr std::size_t skipPathPrefix(const char (&s)[N], std::size_t n =
 
 #if defined __GNUC__ || defined __clang__
 #  define LOG_CONDITIONAL(log,type)                                                                \
-    __builtin_expect((!Log::isShutdownCalled() && log.type()), 0)
+    __builtin_expect((!Log::isShutdownCalled() && /*log.type()*/false), 0)
```

Comment:
Why?

Reply:
The asan output and our own logging interleaved and I disabled the later locally. I forgot to undo that before commit, it's great that you caught it. :-)